Introduction to Kotlin Multiplatform/06_jvm_render.md

```diff
@@ -44,21 +50,24 @@ common library. It opens the ways to re-use the HTML rendering
 code between our JVM server-side and JS client-side.
 
 We may need to refresh the Gradle project model now to make sure
-the IDE sees all our project model changes
+the IDE sees all our project model changes. Let's click on the _refresh_
+icon in the Gradle tab to implement it
 
 ## Moving Code
 
 Sharing code can be done with the following algorithm
```

Comment:
Sharing code can be done with the following algorithm

* move all the source files into the `src/commonMain/kotlin` folder
* replace the missing platform-specific symbols with `expect` declarations
* add the `actual` keyword for the `expect` declarations for all platforms

Now it is time to run the algorithm on our project. We move
the following files from the `src/jvmMain/kotlin` into `src/commonMain/kotlin`:

* `colour.kt`
* `complex.kt`
* `geometry.kt`
* `render.kt`